lib/dpul_collections/indexing_pipeline/figgy/hydration_consumer.ex

```diff
@@ -68,6 +68,38 @@ defmodule DpulCollections.IndexingPipeline.Figgy.HydrationConsumer do
     |> Broadway.Message.put_data(message_map)
   end
 
+  @impl Broadway
+  # pass through messages and write to cache in batcher to avoid race condition
```

Comment:
Can we say more about the race condition in this comment? Or point to somewhere else where it's discussed?

Reply:
Does one of these handle_message clauses handle deletions? Could we add a comment clarifying the patterns matched here?